### PR TITLE
Current situation first version

### PR DESCRIPTION
**Pull request by PixelSupreme**

### CODE REVIEW

File: Thesis.tex [hunk 1]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
 
 \section{Globalisation \& increased complexity of rights management}
 \label{sec:GlobalRightsMgmt}
+The global development of industries over the past few decades had quite a significant impact related to patents. More companies from all over the world hold patents relevant to sometimes whole industry branches. The number of patents involved with modern products and services also reached a new scale. One example for this is the LTE \footcite{Long Term Evolution} standard for mobile communications that lead to several law suits between different technology companies over a number of years.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Companies from all over the world _sometimes_ hold patents relevant to whole industry branches.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: Furthermore, the number of _licensed_ patents _used_ in modern products and services _has taken on a whole new dimension_.

I'm actually not a hundred percent sure if "take on a new dimension" is used correctly in this context. I will ask my english professor on wednesday.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: One example for this is the LTE standard for mobile communcations: Differing opinions and a multitude of affected patents belonging to different technology companies, led to several law suits between these parties and a delay of years until the adaption of this standard.

File: Thesis.tex [hunk 2]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+\subsection{Patent magnitude}
+As technology gets more complex, more companies are involved in development of individual elements that make up standards, services and products of our modern age. And since new technologies are often derived from established ones it is no surprise that some modern products are related to astounding numbers of patents. As an example we can take a look at the 3GPP mobile communications standard. It is estimated that there are over 8000 patents considered essential for this industry standard, where approximately 90\% of those are split between 12 companies \parencite{Wiki3G}. It is clear that with such large pools of patents relevant to industry standards proper licensing is a complex matter.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: And, since new [...] established ones, it is no surprise, that some modern products are related to an astounding numbers of patents.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: As an example, we take a look at the 3GPP mobile communications standard.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: It is clear, that with such large pools of patents relevant to industry standards, proper licensing is a complex matter.

File: Thesis.tex [hunk 3]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+The follow-up standard LTE is also no exception. While it is not easy to find exact numbers on the size of the patent pool related to the standard, its effects were rather visible over the past few years. A number of lawsuits have been filed over standard relevant patents: Huawei versus ZTE \parencite{HuaweiVsZTE}, Apple versus Ericsson \parencite{AppleVsEricsson}, Huawei versus Samsung \parencite{HuaweiVsSamsung} and Huawei versus T-Mobile \parencite{HuaweiVsTmobile} to just name a few.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: The follow-up standard LTE is no exception _either_.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: [...] and Huawei versus T-Mobile, _just to name a few_.

File: Thesis.tex [hunk 4]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+\subsection{Cross-licensing}
+To limit the risk of patent infringement lawsuits many companies enter cross-licensing agreements. Parties involved in a cross-licensing agreement grant each other a license to exploit patents related to the subject matter of the agreement. Very often each party owns patents related to different aspects of a given product. Thus a cross-license maintains their freedom to bring new products to market \parencite{CrossLicense}.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: To limit the risk of patent infringement lawsuits, many companies enter cross-licensing agreements.

File: Thesis.tex [hunk 5]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+A main limitation of cross licensing is that it is ineffective against patent holding companies. Their primary business model is to acquire and license patents for monetary royalty. As such hey have no need to obtain the rights to exercise other companies' patents. Today patent holding companies are often referred to as patent trolls \parencite{PatentTroll}.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Their primary business model is to acquire patents, _and exploit these for monetary royalties_.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: _Thus, they_ have no _benefit in obtaining_ the rights to _use_ other companies' patents.

File: Thesis.tex [hunk 6]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+\subsection{Reasonable and non-discriminatory licensing}
+To manage the sheer magnitude of patents involved in modern standards, a licensing scheme based on reasonable, non-discriminatory terms (sometimes also fair, reasonable and non-discriminator terms) has been adopted by standard setting organizations and accredited by courts.
+
+This refers to a voluntary licensing commitment often requested by standards organisations from holders of patents that might become essential to a technical standard. Fair relates to the underlying license terms, usually based on anti-trust laws to prevent anti-competitive demands of the licensee. Reasonable refers to the hight of license rates. Non-discriminatory relates to both the terms and rates. As the name suggest it means that a licensor treats different licensees under similar terms and rates \parencite{WikiRAND}.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Reasonable refers to the _height_ of license rates.

**Review by KilnOfTheSecondFlame — CHANGES_REQUESTED** [review]

File: Thesis.tex (hunk 1 above)
Comment (NathDrac) [reply Thesis.tex]: <<<< Differing opinions and a multitude of affected patents belonging to different technology **companies led** to several law suits between these parties and a delay of years until the adaption of this standard.

File: Thesis.tex (hunk 2 above)
Comment (NathDrac) [reply Thesis.tex]: Also ich würde kein komma nach clear setzen, aber ganz klar eins vor proper

File: Thesis.tex (hunk 4 above)
Comment (NathDrac) [reply Thesis.tex]: Ich würde hier use patents verwenden anstelle exploit patents. Exploit hat definitiv in der Mehrzahl der Bedeutungen einen negativ wertenden Beigeschmack

File: Thesis.tex (hunk 5 above)
Comment (NathDrac) [reply Thesis.tex]: patent holding companies ist denke ich nicht der richtige Begriff, es sind die NPE (Non Practicing Entities), welche Patente halten, ohne sie selber zu verwenden.

File: Thesis.tex [hunk 7]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+\subsection{Cross-licensing}
+To limit the risk of patent infringement lawsuits many companies enter cross-licensing agreements. Parties involved in a cross-licensing agreement grant each other a license to exploit patents related to the subject matter of the agreement. Very often each party owns patents related to different aspects of a given product. Thus a cross-license maintains their freedom to bring new products to market \parencite{CrossLicense}.
+
+A main limitation of cross licensing is that it is ineffective against patent holding companies. Their primary business model is to acquire and license patents for monetary royalty. As such hey have no need to obtain the rights to exercise other companies' patents. Today patent holding companies are often referred to as patent trolls \parencite{PatentTroll}. 
+
+\subsection{Reasonable and non-discriminatory licensing}
+To manage the sheer magnitude of patents involved in modern standards, a licensing scheme based on reasonable, non-discriminatory terms (sometimes also fair, reasonable and non-discriminator terms) has been adopted by standard setting organizations and accredited by courts.
```
Comment (NathDrac) [inline Thesis.tex]: (sometimes also fair, reasonable and non-discriminator**y** terms)

File: Thesis.tex (hunk 6 above)
Comment (NathDrac) [reply Thesis.tex]: As the name suggest**s,** it means

File: Thesis.tex [hunk 8]
```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
...
+
+This refers to a voluntary licensing commitment often requested by standards organisations from holders of patents that might become essential to a technical standard. Fair relates to the underlying license terms, usually based on anti-trust laws to prevent anti-competitive demands of the licensee. Reasonable refers to the hight of license rates. Non-discriminatory relates to both the terms and rates. As the name suggest it means that a licensor treats different licensees under similar terms and rates \parencite{WikiRAND}. 
+
+Terms for RAND licensing are often based on a decision of the Federal Court of Justice of Germany from 2009. The court ruled that a defendant accused of patent infringement who was not able to obtain a license from the licensee may defend himself by invoking an abuse of a dominant market position \parencite{OrangeBook}.
+ 
+Another important case related to RAND is Microsoft versus Motorola. In 2012 the United States Court of Appeals for the Ninth Circuit ruled that a patent holder's agreement to a policy like RAND creates a legally binding contract \parencite{MicrosoftVsMotorola}.
```
Comment (NathDrac) [inline Thesis.tex]: In 2012**,** the United States Court of Appeals

Comment (NathDrac) [reply Thesis.tex]: Scheint mir sehr gut recherchiert!

**Review by KilnOfTheSecondFlame — APPROVED** [review]

**Pull request merged**